metricflow/sql/render/snowflake.py

```diff
@@ -17,6 +18,33 @@ def visit_generate_uuid_expr(self, node: SqlGenerateUuidExpression) -> SqlExpres
             execution_parameters=SqlBindParameters(),
         )
 
+    def visit_percentile_expr(self, node: SqlPercentileExpression) -> SqlExpressionRenderResult:
+        """Render a percentile expression for Snowflake. Add additional over() syntax for window."""
```

Comment:
This `over` syntax comment looks like the same leftover from earlier work as in the Databricks renderer. Shall we remove it? Or should this be `WITHIN GROUP` instead of `over()`?